common-tasks/managing-appvm-shortcuts.md

```diff
@@ -26,6 +26,53 @@ After that, select the *Add more shortcuts* entry in the VM's submenu to customi
 
 The above image shows that Windows HVMs are also supported (provided that Qubes Tools are installed).
 
+What if my application has not been automatically included in the list of available apps?
+-----------------------------------------------------------------------------------------
+
+Some times applications may not have included a `.desktop` file and may not be detected by `qvm-sync-appmenus`.
+Other times, you may want to make a web shortcut available from the Qubes start menu.
+
+You can manually create new entries in the "available applications" list of shortcuts for all AppVMs based on a TemplateVM.
+To do this:
+
+1. Open a terminal window to the TemplateVM.
+2. Create a custom `.desktop` file in `/usr/local/share/applications` (you may need to first create the subdirectory).
```

Comment:
This will work only if you want to start the application in the TemplateVM itself. `/usr/local` is in `/rw`, so it isn't shared with TemplateBasedVMs. While not ideal from filesystem hierarchy point of view, it needs to be in `/usr/share/applications` to work everywhere.